tensorboard/webapp/hparams/_redux/hparams_selectors.ts

```diff
@@ -76,14 +76,17 @@ export const getHparamFilterMap = createSelector(
   getHparamFilterMapResultFunction
 );
 
-export function getHparamFilterMapFromExperimentIds(experimentIds: string[]) {
+export function getHparamFilterMapFromExperimentIds(
+  experimentIds: string[],
+  includeDefaults?: boolean
```

Comment:
Would it make more sense to add "excludeDefaults"? That way we can remove them where we want but nothing is changed if this boolean is not included?

In fact I am a bit confused how this does not break current uses. Clearly there is something I am not understanding here.